src/accounts.rs

```diff
@@ -325,21 +329,30 @@ impl Config {
             selected_account: 0,
             next_id: 1,
         };
-        let cfg = Config {
-            file: dir.join(CONFIG_NAME),
-            inner,
-        };
+        let file = dir.join(CONFIG_NAME);
+        ensure!(
+            CONFIG_FILES.lock().unwrap().insert(file.clone()),
```

Comment:
These unwrap()s probably isn't a problem, they panic only if the mutex is poisoned, i.e. another thread is panicked while holding the mutex